src/app.js

```diff
@@ -1,299 +1,364 @@
+    .setKey('YOUR_API_KEY');        // Replace with your API Key
+   //.setJWT('jwt');                // Use this to authenticate with JWT generated from Client SDK
+
+const databases = new Databases(client);
+const functions = new Functions(client);
+const storage = new Storage(client);
+const users = new Users(client);
+const account = new Account(client);
+
+let databaseId;
 let collectionId;
 let documentId;
 let userId;
 let bucketId;
 let fileId;
 let functionId;
+let executionId;
 
 // List of API Definitions
 const createCollection = async () => {
-  console.log(chalk.greenBright('Running Create Collection API'));
-
-  const database = new Databases(client, "default");
-
-  const databaseResponse = await database.create("Default");
-  console.log(databaseResponse);
-
-  const response = await database.createCollection(
-    "unique()", // ID of the collection
-    'Movies', // Collection Name
-    "collection", // Marking permission as collection-level
-    ['role:all'], // Read permissions
-    ['role:all'], // Write permissions
-  );
-
-  console.log(response);
-  collectionId = response.$id;
-
-  const nameAttributeResponse = await database.createStringAttribute(collectionId, 'name', 255, false, "Empty Name", false);
-  console.log(nameAttributeResponse);
-
-  const yearAttributeResponse = await database.createIntegerAttribute(collectionId, 'release_year', false, 0, 5000, 1970, false);
-  console.log(yearAttributeResponse);
-
-  console.log("Waiting a little to ensure attributes are created ...");
-  await new Promise((resolve) => setTimeout(resolve, 2000));
+    console.log(chalk.greenBright('Running Create Collection API'));
+
+    const databaseResponse = await databases.create(ID.unique(), "Default");
+
+    databaseId = databaseResponse.$id;
+
+    console.log(databaseResponse);
+
+    const response = await databases.createCollection(
+        databaseId,         // ID of the collection
+        ID.unique(),        // Collection ID
+        "Collection", // Collection Name
+        [
+            Permission.read(Role.any()),
+            Permission.create(Role.users()),
+            Permission.update(Role.users()),
+            Permission.delete(Role.users()),
+        ]
+    );
+
+    console.log(response);
+    collectionId = response.$id;
+
+    const nameAttributeResponse = await databases.createStringAttribute(
+        databaseId,
+        collectionId,
+        'name',
+        255,
+        false,
+        "Empty Name",
+        false
+    );
+    console.log(nameAttributeResponse);
+
+    const yearAttributeResponse = await databases.createIntegerAttribute(
+        databaseId,
+        collectionId,
+        'release_year',
+        false,
+        0, 5000,
+        1970,
+        false
+    );
+    console.log(yearAttributeResponse);
+
+    console.log("Waiting a little to ensure attributes are created ...");
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Add code to create index.

```suggestion
    await new Promise((resolve) => setTimeout(resolve, 2000));

    const yearIndexResponse = await databases.createIndex(
        databaseId,
        collectionId,
        'key_release_year_asc',
        'key',
        ['release_year'],
        ['ASC'],
    );
    console.log(yearIndexResponse);

    console.log("Waiting a little to ensure index is created ...");
    await new Promise((resolve) => setTimeout(resolve, 2000));
```